docs/static/pipeline-configuration.asciidoc

```diff
@@ -16,23 +20,25 @@ output {
 }
```

Comment:
Security on by default. Expand this example to include security? Or replace with a simpler example?

Reply:
++ on making this an elastic cloud example:

```
output {
  elasticsearch { cloud_id => ".." api_key => ".." }
  stdout { codec => rubydebug }
}
```